Makefile

```diff
@@ -5,6 +5,7 @@ BUILDTAGS=seccomp
 export GOPATH:=$(CURDIR)/Godeps/_workspace:$(GOPATH)
 
 all:
+	ln -sfn $(CURDIR) $(CURDIR)/Godeps/_workspace/src/github.com/opencontainers/runc
 	go build -tags "$(BUILDTAGS)" -o runc .
```

Comment:
@LK4D4 I was thinking there might be other use for this symlink, like `make localtest`, so we should keep it before we clean it, since it'll be ignored by git, it won't bother.